src/components/ListOfComments.tsx

```diff
@@ -0,0 +1,67 @@
+import React, {useState} from 'react';
+import Comment from './Comment';
+import CreateComment from './CreateComment';
+import { makeStyles } from '@material-ui/core/styles'
+import {
+    Container, 
+    Typography,
+    Grid,
+    Button,
+    Hidden
+} from '@material-ui/core';
+import comments from '../store/Comments';
+
+
+const useStyles = makeStyles({
+
+    top: {
+        borderStyle: 'solid'
+    },
+    commentText: {
+        fontSize: '19px',
+        fontWeight: 'bold'
+    },
+    createComment: {
+        background: 'lightgrey'
+    },
+
+
+});
+
+
+const ListOfComments: React.FC = () => {
+
+    const classes = useStyles();
+
+    const [createComment, setCreateComment] = useState(true);
+
+    const createCommentText:string = (createComment ? "Create Comment" : "Cancel Comment");
+
+    return(
+        <Container>
+            <Grid container>
+                <Grid item xs={9} className={classes.top}>
+                    <Typography className={classes.commentText}>Comments (999999)</Typography>
```

Comment:
Wait, does this automatically show 999999 comments for each post?

Reply:
yes for now. we will update that in advanced frontend

Reply:
But isn't there a list of comments already, why not take the length of that list?

Reply:
Oh yeah we should lol